src/main/java/seedu/modtrek/MainApp.java

```diff
@@ -7,6 +7,7 @@
 
 import javafx.application.Application;
 import javafx.stage.Stage;
+<<<<<<< HEAD:src/main/java/seedu/modtrek/MainApp.java
```

Comment:
You will have to remove this line. It is the reason the CI checks are failing.

Reply:
Ok